README.md

```diff
@@ -214,6 +213,9 @@ Cluster autoscaling is supported for Windows instances.
 - Create a Windows node through a MachineSet (see spec in [Usage section](https://github.com/openshift/windows-machine-config-operator#usage)).
 - Define and deploy a [MachineAutoscaler](https://docs.openshift.com/container-platform/latest/machine_management/applying-autoscaling.html#configuring-machineautoscaler), referencing a Windows MachineSet.
 
+### Container Runtime
+Windows instances brought up with WMCO are set up with Containerd Container runtime.
```

Comment:
Based on their docs, as well as the [k8s docs](https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd) `containerd` should be all lowercase
additionally i think it might be more correct to say `the containerd container runtime`